docs/user/whats-new.asciidoc

```diff
@@ -1,16 +1,258 @@
+you can add a red line to signify where “now” is on time series visualizations.
+
+[role="screenshot"]
+image::images/highlights-vis-time.png[Visualization showing a red line that indicates the current time, width="75%"]
+
+[discrete]
+==== Collapse by option
+
+Let’s say you’re creating a partition chart with sampled data, such as metrics,
+and want to break down the last values from each reporting dimension, such as hosts.
+You can now use the new *Collapse by* option to break down and summarize the
+result of each last value from hosts without displaying the detailed reporting dimension.
+
+[role="screenshot"]
+image::images/highlights-vis-collapse.png[Collapse by option for breaking down and summarizing the last value, width="75%"]
+
+[discrete]
+[[highlights-8.5-maps]]
+=== Maps &mdash; hide tooltips
+
+Tooltips display when you hover over a document or cluster in a map.
+Don't want to show them? Turn them off with
+the new *Show tooltips* toggle in *Layer settings*.
+
+[role="screenshot"]
+image::https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt553f1d51649d508d/63445a7f2a8d0a11633dbb8f/highlights-maps.gif[Show tooltips control in Layer settings]
+
+
+[discrete]
+[[highlights-8.5-machine-learning]]
+=== Machine Learning
+
+[discrete]
+==== Anomaly detection jobs from Lens
+
+You can now create anomaly detection jobs from the flyout of a Lens visualization.
+Enter a job ID before creating the job, so it starts automatically after saving.
+For a more advanced configuration, follow the link that takes you to the ML job wizard.
+
+[role="screenshot"]
+image::images/highlights-ml-lens.png[Flyout for creating an anomaly detection job from Lens]
+
+[discrete]
+==== Log pattern analysis
+
+You can now select a data view and message field,
+apply optional filters, and then run on-demand categorization analysis.
+Messages that are similar are grouped using the same underlying algorithms
+as a machine learning categorization job. The distribution of the documents in each category
+are highlighted on the main chart. You can then open *Discover*,
+use one or more of these categories as a filter to view the matching documents,
+and continue your investigation.
+
+[role="screenshot"]
+image::images/highlights-ml-log-analysis.png[Chart showing the distribution of documents in each cateogry]
+
+[discrete]
+==== Locations of anomalous results in Maps
+
+This release adds a link to *Maps* in the *Anomaly Explorer* and *Single Metric Viewer*
+for any anomalies where the source data contains geo fields,
+even if the analysis does not use the location field directly.
+View the anomaly on the map by following the link.
+For example, view the location of a client making an unusually high number of
+requests that failed with a ‘page not found’ 404 error.
+
+[role="screenshot"]
+image::https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt6af384c924ab4f13/634453fe4a80ba11acd3c9c1/highlights-ml-lens.gif[Viewing anomalies on a map]
```

Comment:
@lcawl this is a gif that isn't working.  If we can't use it, can you please proved a screenshot that shows the maps link.

Reply:
I'll follow up with @grabowskit but for now I'm fine with omitting it.